src/index.js

```diff
@@ -0,0 +1,112 @@
+import './style.css';
+import {
+  form, todoInput, todosListEl,
+} from './modules/variables.js';
+import showNotification from './modules/notification.js';
+import setTime from './modules/dates.js';
+
+setTime();
+let todos = JSON.parse(localStorage.getItem('todos')) || [];
+let EditTodoId = -1;
+
+/// define renderTodos function
+function renderTodos() {
+  if (todos.length === 0) {
+    todosListEl.innerHTML = '<br><center>Nothing to do!</center>';
+    showNotification('Nothing to Do');
+    return;
+  }
+  todosListEl.innerHTML = '';
+  todos.forEach((todo, index) => {
+    todosListEl.innerHTML += `
+    <div class="todo" id=${index}>
+    <i 
+      class="bi ${todo.checked ? 'bi-check-circle-fill' : 'bi-circle'}"
+      style ="color : ${todo.color} "
+      data-action='check';
+      ></i>
+    <p class="${todo.checked ? 'checked' : ''} " style ="background-color : ${todo.color}" data-action='check' >${todo.value}</p>
+    <i class="bi bi-pencil-square" data-action="edit"></i>
+    <i class="bi bi-trash" data-action="delete"></i>
+  </div>`;
+  });
+}
+
+todoInput.placeholder = 'type to do here';
+todoInput.style.color = 'black';
+todoInput.style.backgroundColor = 'lightgray';
+todoInput.style.borderRadius = '25px';
```

Comment:
I humbly suggest that you create a module which is a class for the todoInput, where this initialization will be encapsulated inside the constructor, this helps you to have [DRY](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/clean-code/lesson_DRY_KISS_YAGNI.md) code and also makes it to scale up your code and add new features i.e drag and drop 😉

Reply:
@tsohleDev 
Kindly suggest how I am supposed to do this.

Reply:
[Issue resolved.](https://github.com/Lucash2022/To-Do-list-review/pull/1/commits/781b03ea9c0bfe433be0380d407f67a066d9a8bf) Thank you